test/pod_performance/README.md

```diff
@@ -0,0 +1,37 @@
+# Kueue Pod Integration Performance Testing
+## Introduction
+A minimal setup for performance testing Plain Pods integration using
+clusterloader2.
+
+## Setup
+
+Install Kueue, configured to use [Plain Pods](https://kueue.sigs.k8s.io/docs/tasks/run_plain_pods/).
+
+e.g.
+```
+wget https://github.com/kubernetes-sigs/kueue/releases/download/v0.6.2/manifests.yaml
+
+# manifest.diff is provided in this directory
+patch manifests.yaml manifest.diff
+kubectl apply --server-side -f manifests.yaml
+```
+
+Install and build clusterloader - see https://github.com/kubernetes-sigs/kueue/tree/main/test/performance#how-to-run-the-test
+Then, set the env variable `CL2_HOME_DIR` to its location.
+
+PROVIDER and KUBECONFIG env variables may also be overridden.
+
+## Running
```

Comment:
Can you add an overview of the variables that users might want to change or provide as environment variables to test different setups?